hack/docker/setup-mongo.sh

```diff
@@ -14,7 +14,8 @@ source "$REPO_ROOT/hack/libbuild/common/public_image.sh"
 
 APPSCODE_ENV=${APPSCODE_ENV:-dev}
 IMG=stash
-RESTIC_VER=${RESTIC_VER:-0.9.4}
+RESTIC_VER=${RESTIC_VER:-0.8.3}
```

Comment:
As database support is only for v1beta1 API, we can use only one `restic` version for database image.